shared/src/types/vote_extensions/validator_set_update.rs

```diff
@@ -186,12 +205,19 @@ impl VotingPowersMapExt for VotingPowersMap {
                 let voting_power = voting_power.round().to_integer();
```

Comment:
I can't add a suggestion to L181, but we should use a permalink there i.e. https://github.com/anoma/ethereum-bridge/blob/fe93d2e95ddb193a759811a79c8464ad4d709c12/test/utils/utilities.js#L29